Makefile

```diff
@@ -0,0 +1,36 @@
+REGISTRY ?= ghcr.io
+OWNER ?= racklet
+IMAGE_NAME ?= drawio-render-action
+IMAGE = $(REGISTRY)/$(OWNER)/$(IMAGE_NAME)
+TAG ?= 
+
+GIT_VERSION=$(shell git describe --tags HEAD 2>/dev/null)
+ifeq ($(shell git status -s),)
+GIT_STATUS=
+else
+GIT_STATUS=-dirty
+endif
+GIT_TAG=$(GIT_VERSION)$(GIT_STATUS)
+GIT_MAJOR=$(shell echo $(GIT_VERSION) | cut -d. -f1)
+
+all: build
+build:
+	docker build -t $(IMAGE):$(GIT_TAG) --pull .
+
+push: build
+	docker push $(IMAGE):$(GIT_TAG)
+ifneq ($(TAG),)
+	docker tag $(IMAGE):$(GIT_TAG) $(IMAGE):$(TAG)
+	docker push $(IMAGE):$(TAG)
+endif
+ifeq ($(GIT_STATUS),)
```

Comment:
split this out to a dedicated `push-major` tag, to not override the "stable" tag for e.g. main branch pushes?

Reply:
Strange; the diff didn't show it correctly really by default, I did it now however, so it'll be included in v1.